### PR TITLE
Release specific doc updates

### PR DESCRIPTION
**Pull request by keirthana**
All release and version specific updates are grouped in this PR. There will be one more commit to this PR once the release notes is finalised. I have a couple more points and links to add to the release notes.

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -3,6 +3,67 @@
...
+* Since the NVIDIA package is being held from upgrading when performing `apt upgrade`, this leads to the drivers missing the security patch. A Juju action is implemented to simplify this manual upgrade and perform the NVIDIA drivers upgrade on demand. <!--AC-1436-->
+* Android security updates for May 2023 (see [Android Security Bulletin - May 2023](https://source.android.com/docs/security/bulletin/2023-05-01) for more information).
+
+## Bug fixes
+
+The following issues were fixed as part of the Anbox Cloud 1.18 release:
+
+* The following timestamp issue was reported by Android framework:
+
+    Choreographer: Frame time is 0.278502 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
```
Comment (adglkh) [inline release-notes.md:54]: Let's fold the above three lines into a single one:
* Fix the incorrect vsync timestamps in anbox hardware composer

**Review by ajanon — CHANGES_REQUESTED** [review]

**Review by adglkh — CHANGES_REQUESTED** [review]

File: component-versions.md [hunk 2]
```diff
@@ -2,6 +2,86 @@ This documents the versions of the different components for each Anbox Cloud rel
...
+### Validated revisions of 3rd party charms
+
+n/a
+
+### Bundles
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud` | `1.18/stable` | 78 |
+| `anbox-cloud-core` | `1.18/stable` | 83 |
```
Comment (morphis) [inline component-versions.md:48]: The revisions will change when we run the `promote-bundles` job today. So these are not final.

File: roadmap.md [hunk 3]
```diff
@@ -14,16 +14,16 @@ A patch release for Anbox Cloud is released at the beginning of every month and
...
 
 * Android security updates for June 2023
 * Bug fixes
+
+### 1.18.2 (July 2023)
+
+Target date: July 20, 2023
+
+* Android security updates for July 2023
+* Bug fixes
```
Comment (morphis) [inline roadmap.md:29]: Can you add the 1.19.0 item too

```
### 1.19.0 (August 2023)

Target date: August 16, 2023

* Pagination / filtering support for parts of the AMS API (containers + applications)
* AV1 hardware accelerated video encoding on NVIDIA GPUs
* OpenAPI based documentation of the AMS API
* Cgroupv1 compatiblity layer to support Android on cgroupv2-only systems
* Vulkan support for NVIDIA GPUs (alpha)

**Review by morphis — CHANGES_REQUESTED** [review]

**Review by adglkh — APPROVED** [review]
LGTM

**Review by ajanon — APPROVED** [review]
LGTM

**Review by adglkh — APPROVED** [review]
LGTM

File: index.md [hunk 4]
```diff
@@ -13,11 +13,12 @@ You should consider using Anbox Cloud if you want to deliver mobile applications
 
 ## What's new
 
-Along with bug fixes and general improvements, Anbox Cloud 1.17 includes the following changes:
+Along with bug fixes and general improvements, Anbox Cloud 1.18 includes the following changes:
 
-* Enablement work of Vulkan support for NVIDIA GPUs
-* Dynamic screen resolution support
-* Start and stop for containers in AMS
+* Vulkan support for NVIDIA GPUs
```
Comment (ajanon) [inline index.md:18]: Wait, isn't that only for 1.19?

**Pull request merged**